tests/nlu/tokenizers/test_convert_tokenizer.py

```diff
@@ -25,55 +31,133 @@
-    )
+def test_custom_intent_symbol(text, expected_tokens):
+
+    component_config = {
+        "name": "ConveRTTokenizer",
+        "model_url": RESTRICTED_ACCESS_URL,
+        "intent_tokenization": True,
+        "intent_split_symbol": "+",
+    }
+
+    tokenizer = ConveRTTokenizer(component_config, ignore_exceptions=True)
 
     message = Message(data={TEXT: text})
     message.set(INTENT, text)
 
-    tk.train(TrainingData([message]))
+    tokenizer.train(TrainingData([message]))
 
     assert [t.text for t in message.get(TOKENS_NAMES[INTENT])] == expected_tokens
 
 
+@pytest.mark.skip_on_windows
 @pytest.mark.parametrize(
     "text, expected_number_of_sub_tokens",
     [("Aarhus is a city", [2, 1, 1, 1]), ("sentence embeddings", [1, 3])],
 )
-@pytest.mark.skip
-def test_convert_tokenizer_number_of_sub_tokens(
-    component_builder, text, expected_number_of_sub_tokens
-):
-    tk = component_builder.create_component_from_class(ConveRTTokenizer)
+def test_convert_tokenizer_number_of_sub_tokens(text, expected_number_of_sub_tokens):
+    component_config = {"name": "ConveRTTokenizer", "model_url": RESTRICTED_ACCESS_URL}
+    tokenizer = ConveRTTokenizer(component_config, ignore_exceptions=True)
 
     message = Message(data={TEXT: text})
     message.set(INTENT, text)
 
-    tk.train(TrainingData([message]))
+    tokenizer.train(TrainingData([message]))
 
     assert [
         t.get(NUMBER_OF_SUB_TOKENS) for t in message.get(TOKENS_NAMES[TEXT])
     ] == expected_number_of_sub_tokens
+
+
+@pytest.mark.skip_on_windows
+def test_raise_no_url():
+
+    component_config = {"name": "ConveRTTokenizer"}
+    with pytest.raises(RasaException) as excinfo:
+        _ = ConveRTTokenizer(component_config)
+
+    assert (
+        """Parameter "model_url" was not specified in the configuration of "ConveRTTokenizer"""
+        in str(excinfo.value)
+    )
+
+
+@pytest.mark.skip_on_windows
+@pytest.mark.parametrize(
+    "model_url, exception_phrase",
+    [
+        (ORIGINAL_TF_HUB_MODULE_URL, "which does not contain the model any longer"),
+        (
+            RESTRICTED_ACCESS_URL,
+            "which is strictly reserved for pytests of Rasa Open Source only",
+        ),
+    ],
+)
+def test_raise_invalid_urls(model_url: Text, exception_phrase: Text):
+
+    component_config = {"name": "ConveRTTokenizer", "model_url": model_url}
+    with pytest.raises(RasaException) as excinfo:
+        _ = ConveRTTokenizer(component_config)
+
+    assert exception_phrase in str(excinfo.value)
+
+
+@pytest.mark.skip_on_windows
+def test_raise_wrong_model_directory():
+
+    with tempfile.TemporaryDirectory() as temp_dir:
+
+        component_config = {"name": "ConveRTTokenizer", "model_url": temp_dir}
+
+        with pytest.raises(RasaException) as excinfo:
+            _ = ConveRTTokenizer(component_config)
+
+        assert "Re-check the files inside the directory" in str(excinfo.value)
+
+
+@pytest.mark.skip_on_windows
+def test_raise_wrong_model_file():
```

Comment:
that's a pytest fixture which gives you a temporary directory. No need for using `tempfile` (same for the others)
```suggestion
def test_raise_wrong_model_file(tmp_path: Path):
```